src/mmw/apps/core/templates/base.html

```diff
@@ -31,9 +31,8 @@
     <div class="map-container">
         {% block map %}
         {% endblock map %}
-    </div>
 
-    <div class="container-fluid top-nav" style="display: none;">
+        <!-- Used on registration/account pages, not on main app -->
```

Comment:
Consider making this a Python template comment instead of an HTML comment so that it doesn't show up in the UI where it isn't used:

![image](https://cloud.githubusercontent.com/assets/1430060/13684897/dedb37be-e6db-11e5-9000-714419ae48e3.png)